packages/api/tests/mock.factories.js

```diff
@@ -2155,10 +2155,12 @@ function fakeNotification(defaultData = {}) {
   const notification_id = faker.datatype.uuid();
   return {
     notification_id,
-    translation_key: `translation_key of notification ${notification_id}`,
+    title: { translation_key: `title_translation_key of notification ${notification_id}` },
+    body: { translation_key: `body_translation_key of notification ${notification_id}` },
     variables: [],
-    entity_id: faker.datatype.uuid(),
-    entity_type: `entity_type of notification ${notification_id}`,
+    ref: {
```

Comment:
We may need to update this to handle the case where the notification is supposed to have a `ref: {url}`, which I believe may be the case here